<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QgsProjectPropertiesBase</class>
 <widget class="QDialog" name="QgsProjectPropertiesBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Project Properties</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget2" >
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="btnGrpMapUnits" >
         <property name="title" >
          <string>Map Units</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radMeters" >
            <property name="text" >
             <string>Meters</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radFeet" >
            <property name="text" >
             <string>Feet</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radDecimalDegrees" >
            <property name="text" >
             <string>Decimal degrees</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QGroupBox" name="btnGrpPrecision" >
         <property name="title" >
          <string>Precision</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radAutomatic" >
            <property name="toolTip" >
             <string>Automatically sets the number of decimal places in the mouse position display</string>
            </property>
            <property name="whatsThis" >
             <string>The number of decimal places that are used when displaying the mouse position is automatically set to be enough so that moving the mouse by one pixel gives a change in the position display</string>
            </property>
            <property name="text" >
             <string>Automatic</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radManual" >
            <property name="toolTip" >
             <string>Sets the number of decimal places to use for the mouse position display</string>
            </property>
            <property name="whatsThis" >
             <string>Sets the number of decimal places to use for the mouse position display</string>
            </property>
            <property name="text" >
             <string>Manual</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxDP" >
            <property name="toolTip" >
             <string>The number of decimal places for the manual option</string>
            </property>
            <property name="whatsThis" >
             <string>The number of decimal places for the manual option</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel3" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>decimal places</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QGroupBox" name="grpDigitizing" >
         <property name="title" >
          <string>Digitizing</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="textLabel1_2_2_2" >
              <property name="text" >
               <string>Snapping Tolerance:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="textLabel1_2_2" >
              <property name="text" >
               <string>Line Width:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="textLabel1_2" >
              <property name="text" >
               <string>Line Colour:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLineEdit" name="leSnappingTolerance" >
              <property name="maximumSize" >
               <size>
                <width>120</width>
                <height>32767</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinDigitisedLineWidth" >
              <property name="minimum" >
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="pbnDigitisedLineColour" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>5</hsizetype>
                <vsizetype>5</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>0</width>
                <height>22</height>
               </size>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QGroupBox" name="titleBox" >
         <property name="title" >
          <string>Project Title</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLineEdit" name="titleEdit" >
            <property name="text" >
             <string>Default project title</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Map Appearance</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1" >
            <property name="text" >
             <string>Selection Color:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="pbnSelectionColour" >
            <property name="minimumSize" >
             <size>
              <width>120</width>
              <height>22</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>21</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Projection</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <widget class="QgsProjectionSelector" name="projectionSelector" />
       </item>
       <item row="0" column="0" >
        <widget class="QCheckBox" name="cbxProjectionEnabled" >
         <property name="text" >
          <string>Enable on the fly projection</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="pbnHelp" >
       <property name="text" >
        <string>&amp;Help</string>
       </property>
       <property name="shortcut" >
        <string>F1</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>94</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonApply" >
       <property name="text" >
        <string>&amp;Apply</string>
       </property>
       <property name="shortcut" >
        <string>Alt+A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>QgsProjectionSelector</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselector.h</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
